Makefile

```diff
@@ -122,6 +122,24 @@ git-setup:
 	$(GIT) config blame.ignoreRevsFile .git-blame-ignore-revs || true
 	@echo "## xNVMe:: git-setup [DONE]"
 
+define format-help
+# run code format (style, code-conventions and language-integrity) on staged changes
+endef
+.PHONY: format
+format:
+	@echo "## xNVMe: format"
+	@pre-commit run
+	@echo "## xNVME: format [DONE]"
+
+define format-all-help
+# run code format (style, code-conventions and language-integrity) on staged and committed changes
+endef
+.PHONY: format-all
+format-all:
+	@echo "## xNVMe: format-all"
+	@pre-commit run --all
```

Comment:
`--all` refers to all hooks, `--all-files` will make it run on unstaged files as well.

Reply:
As discussed elsewhere, it is curious... but it is not changed to ``--all-files``.

Reply:
Fixed.